crates/compilers/src/compilers/zksolc/mod.rs

```diff
@@ -728,7 +729,7 @@ mod tests {
         let out = zksolc().compile(&input).unwrap();
         let (_, mut contracts) = out.split();
         let contract = contracts.remove("LinkTest").unwrap();
-        let bytecode = &contract.evm.unwrap().bytecode.unwrap().object;
+        let bytecode = &contract.0.eravm.unwrap().bytecode.unwrap().object;
```

Comment:
```suggestion
        let bytecode = &contract.eravm.unwrap().bytecode.unwrap().object;
```
There's no wrapper anymore right?